spec.html

```diff
@@ -472,18 +472,27 @@ <h1>ShadowRealm ( )</h1>
 			<emu-alg>
 				1. If NewTarget is *undefined*, throw a *TypeError* exception.
 				1. Let _O_ be ? OrdinaryCreateFromConstructor(NewTarget, *"%ShadowRealm.prototype%"*, « [[ShadowRealm]], [[ExecutionContext]] »).
-				1. Let _realmRec_ be CreateRealm().
-				1. Set _O_.[[ShadowRealm]] to _realmRec_.
-				1. Let _context_ be a new execution context.
-				1. Set the Function of _context_ to *null*.
-				1. Set the Realm of _context_ to _realmRec_.
-				1. Set the ScriptOrModule of _context_ to *null*.
+				1. Perform ? InitializeHostDefinedRealm(). The customizations for creating the global object are to return ? InitializeShadowRealm(_O_).
+				1. Let _context_ be the running JavaScript execution context.
 				1. Set _O_.[[ExecutionContext]] to _context_.
-				1. Perform SetRealmGlobalObject(_realmRec_, *undefined*, *undefined*).
-				1. Perform ? SetDefaultGlobalBindings(_O_.[[ShadowRealm]]).
-				1. Perform ? HostInitializeShadowRealm(_O_.[[ShadowRealm]]).
+				1. Let _realmRec_ be the Realm of _context_.
```

Comment:
@Ms2ger I'm confused about this line, and the title of this PR. Who is creating the realm now?

Reply:
`CreateRealm` was folded into `InitializeHostDefinedRealm` in https://github.com/tc39/ecma262/pull/3139, it creates a realm, customizable global object, and execution context which is pushed to the stack. This may be changed again slightly in https://github.com/tc39/ecma262/pull/3139.

Reply:
I'm also a little confused, and still don't understand the purpose of this PR. For ShadowRealm we need a global object with observably non-exotic behavior. Why can't we just let the 262 pretend it created a regular object to use as global, and have implementations actually implement it differently if they want to have their global object with exotic behavior that is non observable?

Reply:
In https://github.com/tc39/ecma262/pull/3139 I see the following: 
> If the host requires use of an exotic object to serve as _realm_'s global object, then

Can we in any way constraint here that the host is not allowed to use an exotic object for the global object of ShadowRealm? And thus pretend that observably we took the path of the Else and created a regular object for the global?